src/TLuaInterpreter.cpp

```diff
@@ -12494,7 +12494,14 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(green, black, s2 );
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(red, black, s3 );
     }
-
+    
+    if( mpHost->mpConsole && mpHost->mEchoLuaErrors )
+    {
+    	  blue = QColor(80,80,255);
+        mpHost->mpConsole->printDebug(blue, black, QString("[ ERROR ] - ") );
```

Comment:
This will look a lot like the "standardised" messages of form:
* "[ ERROR ] - Something has gone wrong"
* "[ WARN ]  - Something unusual has happened, it isn't necessarily wrong but it might be"
* "[ ALERT ] - Something unusual has happened, and this is to let the user know"
* "[ INFO ]  - Something usual has started to happen but overall the process that causes it hasn't finishing happening yet, OR something unusual has happened and this is some more information about it."
* "[  OK  ]  - Something usual has finished and things generally were all right with it."

but will not follow the same colouration.  If you __want__ to use _that_ system, just create a `QString` with a form:

    tr( "[ ERROR ] - Message beginning with a capital letter and ideally manually formatted to less than 80 characters (with some allowance for %1 type variable expansion)\n"
                    "and if split onto multiple lines with the wrapped lines indented to the first non-space character after the first lines '-' to reflect how it will show on screen\n"
                    "and with a full-stop or other end of sentence punctuation but NO line-feed on the last line (it gets stripped off!)" )
      .arg( "String inserted where-ever %1 is, possible multiple times" );

and then use the nearest available `postMessage( <above string> )` method! :grinning: 

_The colouration/formatting of the messages is handled automagically by code in:
`(void) cTelnet::postMessage( QString msg )`_